spec/features/user_visits_place_spec.rb

```diff
@@ -56,4 +74,22 @@
       end
     end
   end
+
+  describe "that doesn't exist" do
+
+    it 'renders a SORRY page' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.